index.html

```diff
@@ -1,10 +1,131 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-	<meta charset="UTF-8">
-	<title>Document</title>
-<link rel="stylesheet" href="/assets/style.css" /></head>
-<body>
-	<h1>Hi</h1>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <title>Document</title>
+    <link rel="stylesheet" type="text/css" href="scss/vendor/reset.css" />
+
+    <link
+      href="https://fonts.googleapis.com/css?family=Raleway&display=swap"
+      rel="stylesheet"
+    />
+
+    
+  <link rel="stylesheet" href="/assets/style.css" /></head>
+  <body>
+    <div class="container">
+      <div class="start-screen-block">
+        <header>
+          <ul>
+            <li><a href="#">home</a></li>
+            <li><a href="#">portfolio</a></li>
+            <li><a href="#">blog</a></li>
+            <li><a href="#">pages</a></li>
+            <li><a href="#">about us</a></li>
+            <li><a href="#">contact</a></li>
+          </ul>
+        </header>
+        <div class="header-logo">
+          <img src="src/img/logo.png" />
+          <p>Expire</p>
+        </div>
+
+        <div class="header-slider">
+          <div class="arrows">
+            <img src="src/img/left.png" class="left-arrow" />
+            <img src="src/img/right.png" class="right-arrow" />
+          </div>
+          <p class="slider-headline">
+            Expire
+          </p>
+          <img src="src/img/line.png" class="line" />
+          <p class="slider-paragraph">
+            Professionaly designed, carefully made for your enjoyement
+          </p>
+          <div class="slider-buttons">
+            <button class="explore-btn">explore</button>
+            <button class="learn-btn">
+              learn more
+            </button>
+          </div>
+          <div class="slider-dots">
+            <div class="focused"></div>
+            <div class="unfocused"></div>
+            <div class="unfocused"></div>
+          </div>
+        </div>
+      </div>
+      <div class="paralax-section">
+        <div class="paralax-headline">
+          <div class="underline">
+            <h2 class="underline-headline">Paralax section</h2>
+          </div>
+          <p class="underline-text">
+            Exploring the unexplored is a risky business, but a rewarding one.
+          </p>
+        </div>
+        <div class="paralax-quote">
+          <span class="quote-symbol">‘‘</span>
+          <p class="quote">
+            Design is in everything we make, but it's also between those things.
+            It's a mix of craft, science, storytelling, propaganda, and
+            philosophy.
+          </p>
+          <p class="quote-author">— Erik Adigard</p>
+        </div>
+      </div>
+      <div class="commercial-data">
+        <ul class="list-of-data">
+          <li>
+            <div class="chapter">
+              <img src="src/img/user.png" />
```

Comment:
Don't forget to use `alt` for images, even it would be empty.

Reply:
Got it